tests/robot-tests/scripts/run_tests_pipeline.py

```diff
@@ -54,5 +57,7 @@ def run_tests_pipeline():
 
     parser.add_argument("--processes", dest="processes", help="number of processes to run", required=True)
 
+    parser.add_argument("--rerun-failed-suites", dest="rerun_failed_suites", help="rerun any failed suites from a previous run", required=False)
```

Comment:
Does this need a UI test pipeline change to add the extra argument, across all of the UI test tasks in the build pipeline and release pipeline

Reply:
Done already ta

Reply:
Ooh but not in the Release pipeline though... I'll see if it does the trick in the build pipeline first and then apply to the release one if it's all good if that's OK?

Reply:
Yep that's cool. 🙂 

Yeah the release pipeline runs the UI tests in a separate stage after every env deployment stage.